src/ngraph/op/fused/elu.cpp

```diff
@@ -28,16 +28,18 @@
 using namespace std;
 using namespace ngraph;
 
-op::Elu::Elu(const shared_ptr<Node>& data, const shared_ptr<Node>& alpha)
-    : FusedOp("Elu", {data, alpha})
+op::Elu::Elu(const shared_ptr<Node>& data, const double& alpha)
+    : FusedOp("Elu", {data})
+    , m_alpha{alpha}
 {
     constructor_validate_and_infer_types();
 }
 
 NodeVector op::Elu::decompose_op() const
 {
     auto data = get_argument(0);
-    auto alpha_node = get_argument(1);
+    shared_ptr<Node> alpha_node =
+        make_shared<op::Constant>(data->get_element_type(), Shape{}, vector<double>{m_alpha});
```

Comment:
Why double constant depends on input data type? and what if data->get_element_type() returns something like FP16?

Reply:
To perform operations each of the operands must be in the same type.
`op::Constant` perform a `static_cast` on input vector under the hood.